### PR TITLE
CTCP-4085

### PR DESCRIPTION
**Pull request by Reece-Carruthers**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -239,7 +239,7 @@ departure.status.departureDeclarationRejected = Departure declaration rejected
 departure.status.departureDeclarationReceived = Departure declaration received
 departure.status.guaranteeValidationFail = Guarantee rejected
 departure.status.transitDeclarationSent = Transit declaration sent
-departure.status.writeOffNotification = Guarantee written off
+departure.status.writeOffNotification = Movement ended
```
Comment (jackheslop96) [inline conf/messages.en:242]: Is this P4 or P5?

Comment (Reece-Carruthers) [reply conf/messages.en:242]: This is P5, this comes from Ken's content ticket: https://github.com/hmrc/manage-transit-movements-frontend/pull/406

Comment (Reece-Carruthers) [reply conf/messages.en:242]: P4 ones have movement at the start IIRC

File: app/viewModels/P5/departure/DepartureStatusP5ViewModel.scala [hunk 2]
```diff
@@ -73,10 +73,11 @@ object DepartureStatusP5ViewModel {
       discrepancies,
       invalidMRN(),
       releasedForTransit(departureId),
-      goodsNotReleased(),
+      goodsNotReleased(departureId, localReferenceNumber),
       guaranteeRejected(departureId, localReferenceNumber),
       incidentDuringTransit(),
-      goodsBeingRecovered(),
+      goodsBeingRecovered(departureId, localReferenceNumber),
+      goodsBeingRecovered(departureId, localReferenceNumber),
```
Comment (MatthewAlexanderBeck) [inline app/viewModels/P5/departure/DepartureStatusP5ViewModel.scala:80]: Duplicate?

File: app/controllers/testOnly/RecoveryNotificationController.scala [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  view: RecoveryNotificationView
+)(implicit val executionContext: ExecutionContext)
+    extends FrontendController(cc)
+    with I18nSupport {
+
+  def onPageLoad(departureId: String, localReferenceNumber: LocalReferenceNumber): Action[AnyContent] =
+    (Action andThen identify).async {
+      implicit request =>
+        departureP5MessageService.filterForMessage[IE035Data](departureId, GoodsBeingRecovered) map {
+          case Some(ie035Data) => Ok(view(viewModelProvider.apply(ie035Data.data), localReferenceNumber))
```
Comment (MatthewAlexanderBeck) [inline app/controllers/testOnly/RecoveryNotificationController.scala:48]: Can we use getMessageById please as its a lttle safer

**Pull request merged**